salt/edx/prod.sls

```diff
@@ -34,11 +34,10 @@ configure_python_ppa_for_edx:
 install_os_packages:
   pkg.installed:
     - pkgs:
-        - git
```

Comment:
I found the package in the ppa here: https://launchpad.net/~git-core/+archive/ubuntu/ppa

the `1:` piece is not necessary, right? Maybe silly, but I just don't know what it means. :)